docs/cluster_configuration.md

```diff
@@ -166,46 +166,72 @@ status:
     policyName: v1-perf-configuration-policy
-    policyNamespace: ztp-sno-ran-du-v4-16
+    policyNamespace: ztp-sno-ran-du-v4-Y-Z
     remediationAction: enforce
   - compliant: Compliant
     policyName: v1-sriov-configuration-policy
-    policyNamespace: ztp-sno-ran-du-v4-16
+    policyNamespace: ztp-sno-ran-du-v4-Y-Z
     remediationAction: enforce
   - compliant: Compliant
     policyName: v1-subscriptions-policy
-    policyNamespace: ztp-sno-ran-du-v4-16
+    policyNamespace: ztp-sno-ran-du-v4-Y-Z
     remediationAction: enforce
 ```
+### Updates to the ClusterInstance defaults ConfigMap
+We assume a ManagedCluster has been installed through a `ProvisioningRequest` referencing the [sno-ran-du.v4-Y-Z-1](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v4-Y-Z-1.yaml) `ClusterTemplate` CR.
 
-### Updates to an existing ACM PolicyGenerator manifest
-
-For updating a manifest in an existing ACM PolicyGenerator, the following steps need to be taken (we'll take [sno-ran-du-pg-v4-16-v1](samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v1.yaml) for examples):
-1. Upversion the cluster template content:
-    * Create a new version of the ACM PG - [sno-ran-du-pg-v4-16-v2](samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v2.yaml):
-        * The name is updated to `sno-ran-du-pg-v4-16-v2` (the `ztp-sno-ran-du-v4-16` namespace is kept).
-        * `policyDefaults.placement.labelSelector.sno-ran-du-policy` is updated from `v1` to `v2` such that the policy binding is updated.
-        * All policy names are updated from `v1` to `v2` (example: `v1-subscriptions-policy` -> `v2-subscriptions-policy`).
-        * The desired manifest section is updated. In the current samples [sno-ran-du-pg-v4-16-v2](samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v2.yaml) adds a `sysctl` section to the `TunedPerformancePatch` section under the `v2-tuned-configuration-policy` policy.
-    * Create a new version of the [clusterinstance-defaults-v1](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v1.yaml) - [clusterinstance-defaults-v2](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v2.yaml):
+In this example we are adding a new route to the [clusterinstance-defaults-v1](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v1.yaml) `ConfigMap` holding default values for the corresponding `ClusterInstance`.
+The following steps need to be taken:
+1. Upversion the cluster template:
+    * Create a new version of the [clusterinstance-defaults-v1](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v1.yaml) `ConfigMap` - [clusterinstance-defaults-v2](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v2.yaml):
         * Update the name to `clusterinstance-defaults-v2` (the namespace stays `sno-ran-du-v4-16`).
```

Comment:
You updated the namespace to "ztp-sno-ran-du-v4-Y-Z" above - should this be the same?

Reply:
Yes, but the namespace should actually be `sno-ran-du-v4-Y-Z` for the ClusterTemplate and the associated resources and `ztp-sno-ran-du-v4-Y-Z` for the policy configuration resources.